src/Security/Security.php

```diff
@@ -655,7 +655,7 @@ public static function clearSessionMessage()
      *
      * @param null|HTTPRequest $request
      * @param int $service
-     * @return HTTPResponse|string Returns the "login" page as HTML code.
+     * @return HTTPResponse|RequestHandler|DBHTMLText|string Returns the "login" page as HTML code.
```

Comment:
Updated PHPDocs in this class match return type hint for `delegateToMultipleHandlers()` which is the value all of these methods return.